web/sites/default/config/core.entity_view_mode.taxonomy_term.ief_table.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 0261fe8c-401f-4828-8b23-c9ac4d6347e7
+langcode: en
+status: true
+dependencies:
+  module:
+    - taxonomy
+id: taxonomy_term.ief_table
+label: 'Inline Entity Form Table'
+targetEntityType: taxonomy_term
+cache: true
```

Comment:
Why is this file part of the PR?

Reply:
Cleaning up a config conflict and went ahead and included it. :)